uk.co.compucorp.civicrm.hrcontactactionsmenu/templates/CRM/ContactActionsMenu/Page/Inline/ActionsPart2.tpl

```diff
@@ -35,7 +35,10 @@
             <button class="button__delete btn-warning">Disable User Account</button>
           </div>
           <div class="col-md-6">
-            <button class="button__disable btn-danger">Delete User Account</button>
+            <button class="button__disable btn-danger" 
+              data-delete-user-url="civicrm/contactactionsmenu/deleteuseraccount">
```

Comment:
You have mentioned `This attribute can be changed to add contact Id form backend`, but dont we need to add it here?

Reply:
We need, but since I am only creating static template, it should be added while integrating this template with backend.